src/main/java/li/cil/oc2/common/item/AbstractStorageItem.java

```diff
@@ -49,6 +51,8 @@ public static ItemStack withCapacity(final ItemStack stack, final int capacity)
     public AbstractStorageItem(final Properties properties, final int defaultCapacity) {
         super(properties);
         this.defaultCapacity = defaultCapacity;
+
+        if(FMLEnvironment.dist == Dist.CLIENT)
         ItemModelsProperties.registerProperty(this, CAPACITY_PROPERTY, AbstractStorageItem::getCapacityProperty);
```

Comment:
I'm guessing this was to keep the number of changes small, but unless they're single-line I like my if bodies as explicit blocks. I.e. with `{}`s around them.

I also pushed an EditorConfig earlier. That *should* introduce a space between the `if` and the `(` when auto formatting. If it doesn't, please let me know!

Sorry you have to be my test-case for code style nitpickery..